include/packager/live_packager_export.h

```diff
@@ -94,10 +94,10 @@ size_t livepackager_buf_size(LivePackagerBuffer_t buf);
 
 typedef struct LivePackager_instance_s* LivePackager_t;
 
-typedef struct LivePackagerStatus_s* LivePackagerStatus_t;
-void live_packager_status_free(LivePackagerStatus_t self);
-
-const char* status_get_error(LivePackagerStatus_t self);
+typedef struct LivePackagerStatus_s {
+  const char* status_message;
```

Comment:
Could you please add a comment here that the client needs to free this memory if it's not NULL?